src/components/menu/AddressInfo/index.tsx

```diff
@@ -1,22 +1,31 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useIntl } from 'react-intl';
 import { useWeb3React } from '@web3-react/core';
 import classNames from 'classnames';
-import { DropdownWrapper, rgba, textCenterEllipsis, useThemeContext } from '@aave/aave-ui-kit';
+import makeBlockie from 'ethereum-blockies-base64';
+import {
+  DropdownWrapper,
+  gradient,
+  rgba,
+  textCenterEllipsis,
+  useThemeContext,
+} from '@aave/aave-ui-kit';
 
 import { useUserWalletDataContext } from '../../../libs/web3-data-provider';
 import { useMenuContext } from '../../../libs/menu';
+import { getNetworkConfig } from '../../../helpers/config/markets-and-network-config';
+import { useStaticPoolDataContext } from '../../../libs/pool-data-provider';
```

Comment:
surprised we arent using some root import file tool

Reply:
Yeah, we should probably setup absolute imports and use them everywhere

Reply:
Created #207 for this